tidb-cloud/migrate-incremental-data-from-mysql.md

```diff
@@ -0,0 +1,250 @@
+        GTID:b631bcad-bb10-11ec-9eee-fec83cf2b903:1-640
+
+Finished dump at: 2022-05-24 11:19:53
+```
+
+Based on the above starting point information, create a migration task as follows.
+
+```toml
+## ********* Task Configuration *********
+name: test-task1
+# shard-mode: "pessimistic"
+# Task mode. The "incremental" mode only performs incremental data migration.
+task-mode: incremental
+# timezone: "UTC"
+
+## ******** Data Source Configuration **********
+## (Optional) If you need to incrementally replicate data that has already been migrated in the full data migration, you need to enable the safe mode to avoid the incremental data migration error.
+##  This scenario is common in the following case: the full migration data does not belong to the data source's consistency snapshot, and after that, DM starts to replicate incremental data from a position earlier than the full migration.
+syncers:        	# The running configurations of the sync processing unit.
+  global:       	# Configuration name.
+  safe-mode: true   # If this field is set to true, DM changes INSERT of the data source to REPLACE for the target database, and changes UPDATE of the data source to DELETE and REPLACE for the target database. This is to ensure that when the table schema contains a primary key or unique index, DML statements can be imported repeatedly. In the first minute of starting or resuming an incremental migration task, DM automatically enables the safe mode.
+
+mysql-instances:
+  - source-id: "mysql-replica-01"
+	block-allow-list:  "bw-rule-1"
+	route-rules: ["route-rule-1"]
+	filter-rules: ["tpcc-filter-rule"]
+	syncer-config-name: "global"                	# You can use the syncers incremental data configuration above.
+	meta:       	                                # When task-mode is "incremental" and the target database does not have a checkpoint, DM uses the binlog position as the starting point. If the target database has a checkpoint, DM uses the checkpoint as the starting point.
+  	binlog-name: "mysql-bin.000001"
+  	binlog-pos: 77092852
+  	binlog-gtid: "b631bcad-bb10-11ec-9eee-fec83cf2b903:1-640"
+
+## ******** Configuration of the Target TiDB on TiDB Cloud **********
+target-database:   	# the Target TiDB on TiDB Cloud
+  host: "tidb.70593805.b973b556.ap-northeast-1.prod.aws.tidbcloud.com"
+  port: 4000
+  user: "root"
+  password: "oSWRLvR3F5GDIgm+l+9h3kB72VFWBUwzOw=="     	# If the password is not empty, it is recommended to use a dmctl-encrypted cipher
+
+## ******** Function Configuration **********
+block-allow-list:
+  bw-rule-1:
+	do-dbs: ["~^tpcc.*"]
+
+routes:                       	# Table renaming rules ('routes') from upstream to downstream tables, in order to support merging different tables into a single target table.
+  route-rule-1:               	# Rule name.
+	schema-pattern: "tpcc"    	# Rule for matching upstream schema names. It supports the wildcards "*" and "?".
+	target-schema: "tpcc"     	# Name of the target schema.
```

Comment:
useless if the schema-pattern and target-schema are the same .

Reply:
ok, changed to tpdd